buildspec.yml

```diff
@@ -0,0 +1,134 @@
+version: 0.2
+
+env:
+  variables:
+    FRAMEWORK_VERSION: '1.3.0'
+    CPU_FRAMEWORK_BINARY: 'https://files.pythonhosted.org/packages/71/64/49c5125befd5e0f0e17f115d55cb78080adacbead9d19f253afd0157656a/mxnet-1.3.0.post0-py2.py3-none-manylinux1_x86_64.whl'
+    CPU_PY_VERSION: '2'
+    CPU_INSTANCE_TYPE: 'ml.c4.xlarge'
+    GPU_FRAMEWORK_BINARY: 'https://files.pythonhosted.org/packages/52/8b/3f9cfe199e592b0723617e5d1919b83022f05cc359bd1885f7d1e9ce4758/mxnet_cu90mkl-1.3.0.post0-py2.py3-none-manylinux1_x86_64.whl'
+    GPU_PY_VERSION: '3'
+    GPU_INSTANCE_TYPE: 'ml.p2.xlarge'
+    ECR_REPO: 'sagemaker-test'
```

Comment:
should we call this sagemaker-pull-request-test?

Reply:
before it's called sagemaker-pull-request-test-integ. @jesterhazy felt it's too long and I made the change. It's in previous comments.